src/components/OSCALCatalogGroups.js

```diff
@@ -52,32 +56,73 @@ const ComponentTab = styled(Tab)(({ theme }) => ({
   textTransform: "none",
 }));
 
-function a11yProps(index, title) {
+function a11yProps(groupId) {
   return {
-    id: `vertical-tab-${index}`,
-    "aria-controls": `vertical-tabpanel-${title}`,
+    id: `vertical-tab-${groupId}`,
+    "aria-controls": `vertical-tab-${groupId}`,
   };
 }
 
 TabPanel.propTypes = {
-  index: PropTypes.number.isRequired,
-  value: PropTypes.number.isRequired,
+  value: PropTypes.string.isRequired,
 };
 
+function determineControlExists(groups, controlLayers, rootLayer) {
```

Comment:
Can we add some js doc to this function? It seems to do a lot and admittedly it is not super obvious what it means to determine a control exists. 

Based on the naming, it seems like the return should be a boolean but it returns what I am guessing is the top level control?